src/System.Management.Automation/engine/ManagementObjectAdapter.cs

```diff
@@ -880,8 +889,9 @@ internal static string GetMethodDefinition(MethodData mData)
         /// <typeparam name="T">PSMemberInfo</typeparam>
         /// <param name="wmiObject">Object for which the members need to be retrieved.</param>
         /// <param name="members">Method information is added to this.</param>
+        /// <param name="predicate"></param>
         protected abstract void AddAllMethods<T>(ManagementBaseObject wmiObject,
-            PSMemberInfoInternalCollection<T> members) where T : PSMemberInfo;
+            PSMemberInfoInternalCollection<T> members, Func<T, bool> predicate = null) where T : PSMemberInfo;
```

Comment:
Above I meant that in such places we could use default predicate:
```suggestion
            PSMemberInfoInternalCollection<T> members, Func<T, bool> predicate = DefaultPredicate) where T : PSMemberInfo;
```